math-2020.html

```diff
@@ -0,0 +1,632 @@
+
+        <p>Additional technical coordination with the following Groups will be made, per the <a href="https://www.w3.org/Consortium/Process/#WGCharter">W3C Process Document</a>:</p>
+
+        <section>
+          <h3 id="w3c-coordination">W3C Groups</h3>
+          <dl>
+	    <dt><a href="https://www.w3.org/Style/CSS/">CSS Working Group</a></dt>
+	    <dd>The Math Working Group will contribute to and coordinate work with the CSS Working Group related to aligning the development and testing of new and existing CSS functionalities that are of interest to the MathML community. The group will designate liaisons to work with the CSS Working Group on issues as needed. Liaisons will ideally be participants of both groups.</dd>
+	    <dt><a href="https://www.w3.org/community/wicg/">Web Incubator Community Group (WICG)</a></dt>
+	    <dd>The Web Platform Incubator Community Group
+	      (<a href="https://www.w3.org/community/wicg/">WICG</a>)
+	      provides a lightweight venue for proposing and
+	      discussing new web platform features, like new HTML
+	      features or Web Platform APIs. Also, new technical
+	      features arising during the development of MathML may
+	      have relevance for the Web Platform in general, or ideas
+	      developing for the Web Platform in general might be
+	      developing in
+<a href="https://www.w3.org/community/wicg">WICG</a>
+which could be relevant to MathML.  The MathML WG will make an effort to make sure these are explored further together within incubations in the Web Platform Incubator Community Group where appropriate.
+	    </dd>
+
+	    <dt><a href="https://www.w3.org/WAI/APA/">Accessible Platform Architectures Working Group</a></dt>
+	    <dd>The APA WG provides horizontal review of potential accessibility issues in the architecture, including knowledge domains for STEM accessibility, as well as of the accessibility impact of a specification. The MathML WG will request reviews and coordinate, and may nominate liaisons to ensure that these groups are aligned and working in concert toward accessible mathematics.</dd>
+
+	    <dt><a href="https://www.w3.org/WAI/ARIA/">ARIA Working Group</a></dt>
+	    <dd>The ARIA Working Group is responsible for the development of ARIA and related Accessibility Mappings. The MathML WG will appoint a liaison and coordinate with the ARIA Working Group to provide the
+<a href="">MathML AAM</a>
+and continue to improve the accessibility of that which is part of MathML.
+</dd>
+
+	    <dt><a href="https://www.w3.org/publishing/groups/publ-wg/">Publishing Working Group</a></dt>
+	    <dd>The Publishing Working Group's goal is to enable all publications to be first class citizens of the web. MathML is used by a large number of technical and textbook publishers in their workflows. The MathML WG will work with the PWG to ensure their needs are met with updates to the MathML recommendations.</dd>
+
```

Comment:
Several comments on this one:
- the Publishing Working Group (renamed Audiobooks Working Group) is now in maintenance mode with a different scope. I am not sure it is worth listing it here.
- instead, I would propose to add the EPUB Working Group to the lot (https://www.w3.org/publishing/groups/epub-wg/). MathML is of a vital importance for the digital publishing world in general, and EPUB in particular. I would actually think it would be useful to have some sort of an active relationship (try to have common WG members, etc) to ensure an active cooperation.
- I would also add the Publishing Community Group (https://www.w3.org/community/publishingcg/) to the list; that group, as a CG, does not have a strong charter control over its work, and is looking at digital publishing issues at large. MathML is a major consideration for that CG as well.

Reply:
Dropping the (now) AudioBooks WG from the groups we will coordinate with and adding the EPUB WG makes a lot of sense. I've fielded a number of questions and comments from them in the past, so I have first hand experience they are interested in what we are doing. Adding the Publishing CG is also sensible. So +1 to @iherman's suggestions.

Reply:
@NSoiffer thanks.

Note that this branch has been merged, so the file is edited elsewhere now... I am not sure how that will be handled.

@bert-github @wseltzer